lib/mparticle_flutter_sdk.dart

```diff
@@ -97,9 +97,13 @@ class MparticleFlutterSdk {
     if (productActionType != null) {
       commerceEvent['productActionType'] =
           ProductActionType.values.indexOf(productActionType);
+      commerceEvent['productActionName'] =
+          productActionTypeString(productActionType);
     } else if (promotionActionType != null) {
       commerceEvent['promotionActionType'] =
           PromotionActionType.values.indexOf(promotionActionType);
+      commerceEvent['promotionActionName'] =
+          promotionActionTypeString(promotionActionType);
```

Comment:
Since this is specific to Android, I think it's best that we don't add complexity to the Dart layer, and remove these lines, and add the switch statements over in the Android layer.  These are actually passed as integer values, so I did this on web. Check out that PR, filename `lib/src/web_helpers/ecommerce_helpers.dart`.   Being part of mapping behavior adds to why I think it should live in the respective platform's code. 

Thoughts?

Reply:
I was thinking that one point,but I noticed two things that lead me to decide to put it here 
1) consistent with how we handled IdentityType
2) going off the ios-specific `Int` value is kind of confusing and error prone...writing a switch statement in the Android for `case 1: ... case 2: .... case 3: ....` is less intuitive than keeping the switch statement in the Flutter layer `AddToCart: ...` `RemoveFromCart: ...` `case Checkout: ...`